Sources/Extensions/NotificationContent/MapViewController.swift

```diff
@@ -89,7 +89,14 @@ class MapViewController: UIViewController, NotificationCategory, MKMapViewDelega
 
         mapView.accessibilityIdentifier = "notification_map"
 
-        let span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        let span: MKCoordinateSpan
+        if let latitudeDelta = CLLocationDegrees(templateValue: haDict["latitude_delta"]),
+           let longitudeDelta = CLLocationDegrees(templateValue: haDict["longitude_delta"]) {
+            span = MKCoordinateSpan(latitudeDelta: latitudeDelta, longitudeDelta: longitudeDelta)
+        } else {
+            span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+        }
+        
```

Comment:
It's hard to make GitHub code suggestions do what's expected here, but this line shouldn't have any spaces on it. You can run `bundle exec fastlane autocorrect` to fix it, or edit the line manually.

Reply:
Done!